docs/configuration/interfaces/index.rst

```diff
@@ -21,7 +21,6 @@ Interfaces
    tunnel
    vti
    vxlan
-   wireguard
```

Comment:
i understand your point, but in the cli wireguard is under interfaces. So at the moment its must belong here.
See also a text about this [here](https://docs.vyos.io/en/latest/documentation.html#configuration-mode-pages)

Reply:
Sure